### ISSUE TITLE
adding delays?

### ISSUE THREAD

**Issue by PolynomialDivision** [xdp-project/xdp-tutorial#197]
Is it possible to add a delay to packages? So I can just replace `tc` and `netem`?

**Comment by tohojo** [comment]
Nope. XDP can't queue packets at all (other than the bulking happening inside the redirect maps). We want to add this, and this is a topic dear to my heart as well, but it'll take a little while to get there...

**Comment by PolynomialDivision** [comment]
Nice. :)
XDP on egress traffic is also still in proposal phase, or?

**Comment by tohojo** [comment]
Polynomdivision <notifications@github.com> writes:

> Nice. :)
> XDP on egress traffic is also still in proposal phase, or?

Yeah, I'd say that the "proper" egress hook is more of a gleam in
@netoptimizer's eye than it's in the "proposal phase" at this point ;)
